ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/AbstractTaskFragment.kt

```diff
@@ -120,7 +121,20 @@ abstract class AbstractTaskFragment<T : AbstractTaskViewModel> : AbstractFragmen
 
   protected fun addSkipButton() =
     addButton(ButtonAction.SKIP)
-      .setOnClickListener { onSkip() }
+      .setOnClickListener {
```

Comment:
Can we extract this event handler to a fun?

Reply:
Resolved, went with onSkip()